packages/components/tag/src/tag-body.tsx

```diff
@@ -45,16 +46,17 @@ const getClickableContentWrapperStyles = (type: TTagBodyProps['type']) => {
 
 const getTextDetailColor = (isDisabled: TTagBodyProps['isDisabled']) => {
   if (isDisabled) return designTokens.fontColorForTagWhenDisabled;
-  return designTokens.fontColorForTag;
+  return designTokens.colorSolid;
 };
 
-const getContentWrapperStyles = (props: TTagBodyProps) => {
+const GetContentWrapperStyles = (props: TTagBodyProps) => {
```

Comment:
I don't think we should use capital letters for the first one since this is not a component.

Reply:
updated here -> [89a39d0](https://github.com/commercetools/ui-kit/pull/2375/commits/89a39d018f49c9437d2b5ff3e70f76e0c4fb75a7)